desktop/mac/release-notes/index.md

```diff
@@ -33,6 +33,46 @@ Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/pro
 > chip](https://desktop.docker.com/mac/main/arm64/Docker.dmg?utm_source=docker&utm_medium=webreferral&utm_campaign=docs-driven-download-mac-arm64){:
 > .button .primary-btn }
 
+### New
+
+- Released Docker Desktop for Linux.
+- Beta release of Docker Extensions and Extensions SDK.
+- New Docker Homepage: Run popular images and discover how to use them.
+- [Compose V2 has now reached GA](https://www.docker.com/blog/announcing-compose-v2-general-availability/)
+
+### Upgrades
+
+- [Compose v2.5.0](https://github.com/docker/compose/releases/tag/v2.5.0)
+- [Go 1.18.1](https://golang.org/doc/go1.18)
+
+### Bug fixes and minor changes
+- Docker Desktop now uses the operating system proxy settings whe available on MacOS and Windows.
+- Fixed a bug that showed Remote Repositories in the Dashboard when running behind a proxy.
+- Fix vpnkit establishing and blocking client connection even if the server is gone. See [docker/for-mac#6235](https://github.com/docker/for-mac/issues/6235).
+- Improvements on volume tab in the Docker Desktop Dashboard:
+  - Show volume size.
+  - Columns can be resized, hidden and reordered.
+  - Columns sort order and hidden state is persisted, even after Docker Desktop restarts.
+  - Row selection is persisted when switching between tabs, even after Docker Desktop restarts.
+- Fixed a bug in the Dev Environments tab that did not add a scroll when more items were added in the screen.
+- Standardise the header title and action in the dashboard screens.
+- Add support for downloading Registry Access Management policies through HTTP proxies.
+- Fixed an issue related to empty remote repositories when the machine went on sleep for long time.
+- Fixed a bug were dangling images were not selected in the cleanup process if their name was not marked as "<none>" but their tag is.
+- Docker Desktop's icon now matches Big Sur Style guide. See [docker/for-mac#6235](https://github.com/docker/for-mac/issues/5536).
```

Comment:
The issue number shown in the link is wrong. And we now have a second PR merged addressing another Dock icon issue, that's why I added it here.

```suggestion
- Docker Desktop's icon now matches Big Sur Style guide. See [docker/for-mac#5536](https://github.com/docker/for-mac/issues/5536).
- [mac] Fixed a problem with duplicate Dock icons and Dock icon not working as expected. Fixes [docker/for-mac#6189](https://github.com/docker/for-mac/issues/6189).
```